packages/e2e-test-kit/src/project-runner.ts

```diff
@@ -29,19 +29,24 @@ export class ProjectRunner {
         before: MochaHook,
         afterEach: MochaHook,
         after: MochaHook,
-        watch = false
+        watch = false,
+        watchedDir = ''
     ) {
         const disposeAfterEach: Set<() => void> = new Set();
         if (watch) {
             const projectToCopy = runnerOptions.projectDir;
-            const tempDir = createTempDirectorySync('local-test');
-            tempDir.path = nodeFs.realpathSync(tempDir.path);
-            const projectPath = join(tempDir.path, 'project');
-            disposeAfterEach.add(tempDir.remove);
+            if(watchedDir){
```

Comment:
watchedDir is a string. what if the target folder doesn't exist? rmdirSync fails in that case, even wih recursive.

Reply:
I think I didn't saw it is because it's true for deep structure and not the first level of existing one.